src/content/docs/tutorial-infrastructure-monitoring/explore-host-data.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Respond to a crashed app
+metaDescription: Solve system outages by investigating resource bottlenecks across your system infrastructure. 
+---
+
+import hostSummaryPage from 'images/infrastructure_screenshot-full_hosts-summary-page.webp'
+
+import metrics1 from 'images/infrastructure_screenshot-full_metrics-time1.webp'
+
+import metrics2 from 'images/infrastructure_screenshot-full_metrics-time2.webp'
+
+import summaryTableAlerting from 'images/infrastructure_screenshot-full_summary-table-alerting.webp'
+
+The performance of your infrastructure determines how well your APIs, services, and apps can make and respond to requests. A hardware failure can prevent your server from handling data, or maybe a memory leak brings an otherwise performant app to an abrupt halt. 
```

Comment:
`determines` is a bold option; maybe `can impact`?

Reply:
Good catch, ty!